infra/filter.lua

```diff
@@ -114,6 +119,13 @@ function Div(el)
     local div = pandoc.CodeBlock( io.read('a'))
     div.classes = el.classes
     return div
+  elseif config.print and #el.classes > 0 and el.classes[1] ~= "center" then
```

Comment:
Is this "if print and not centered, then use bookblock to do left-aligned layout"?

Reply:
I rewrote this code, it was overly obscure. But this is the code that says "for all other blocks, in print mode, use bookblock to make them show up like a colored box.

Reply:
Ok, what's up with the classes != center part then? Why is that needed?

Reply:
Eliminated now